pipeline/steps.go

```diff
@@ -109,7 +115,7 @@ func updateObject(obj Object, data *Context) Result {
 
 func specAndMetaEqual(a, b Object) bool {
 
-	if !equality.Semantic.DeepEqual(a, b) {
+	if !equality.Semantic.DeepEqual(a.GetMeta(), b.GetMeta()) {
 		return false
 	}
```

Comment:
The func is named `specAndMetaEqual` but now it only seems to compare the metadata? Is this correct?
Maybe worth testing? (if not already done)

Reply:
No the spec is compared below.

Reply:
argh ofc :see_no_evil: :tomato: